src/components/Checkbox/Checkbox.module.css

```diff
@@ -1,5 +1,3 @@
-@import '../../design-tokens/mixins.css';
-
```

Comment:
I wonder if we should take this out of the `plop-templates/Component/Component.module.css.hbs` file since it's hanging out in a bunch of files that don't used mixins 🤔

Reply:
https://github.com/chanzuckerberg/edu-design-system/pull/1247

Reply:
Thanks for the cleanup :D